tests/edge-cases/logand-05.txt

```diff
@@ -0,0 +1,2 @@
+brun '(logand (q (1 2)) (q (1 2)))'
+FAIL: logand requires int args ((q 2) (q 2))
```

Comment:
This error message would be hard to understand if I hadn't spent so much time with clvm. I know this is not new in your patch.

Reply:
Maybe we should print just the first invalid argument instead of all of them.